reporting-guide.md

```diff
@@ -440,26 +443,35 @@ Here, I'm requesting `evar1`. In this example, the evar is simply storing a nume
 }
 ```
 
-Let's look in detail at one of the records returned:
-```json
-{
-   "itemId":"743855946",
-   "value":"10",
-   "data":[
-      2032.0
-   ]
-},
-```
+## Filtering Reports
 
-* `itemId` - This is the unique ID associated with this particular value - in this case, campaign "10".
-* `value` - This contains the value of the evar.
-* `data` - This is an array of counts - one for each metric requested.
+Use filters to limit the data returned so that reports show only the values you need.  For example, if you have thousands of records but only a few have needed reports, you can use filtering to return and find them quickly. Some filters also allow you to include, group, or present data in convenient formats, such as breakdown reports.
 
-#### Filters
-In the above example, only the top 5 records were requested and there were 8 pages worth of data available -- only 40 records. What if there were thousands of records, and only a handful were being reported on? Filters can be used to only include those few valuable values in the report.
+This section includes information on the following filtering features:
 
-For the next example, let's only request data for campaigns "10" and "11" by using their itemIds.
-```json
+* [Search parameter](#search)
+
+* [Clause parameter](#clause)
+
+* [Applying Segments](#segments)
+
+* [Breakdown Reports](#breakdown)
+
+### <a name="search">Using `search` Parameters</a>
+
+By using the `search` parameter, you can add even more granular parameters to filter your results more narrowly. The `search` parameter includes the following options:
+
+* `itemId` - A single ID to include in the report
+* `itemIds` - A list of itemIds to include in the report (shown in the example)
+* `excludeItemIds` - A list of itemIds to exclude in the report
+* `clause` - A search clause to use when filtering dimensions
+* `includeSearchTotal` - Includes a special element called 'searchTotals' in the response that contains the total of the filtered items. The default is `false`.
+
+#### Example `search` Request</a>
```

Comment:
This line has a broken anchor tag

Reply:
Thanks for catching that. It's just a stray end tag that I will delete.